v3/testvectors/compatibility_test.go

```diff
@@ -20,13 +20,14 @@ import (
 	s3cryptoV2 "github.com/aws/aws-sdk-go/service/s3/s3crypto"
 	"io"
 	"log"
+	"math"
 	"os"
 	"testing"
 )
 
-const defaultBucket = "s3-encryption-client-v3-go-us-west-2"
+const defaultBucket = "s3ec-go-github-test-bucket"
 const bucketEnvvar = "BUCKET"
-const defaultAwsKmsAlias = "arn:aws:kms:us-west-2:657301468084:alias/s3-encryption-client-v3-go-us-west-2"
+const defaultAwsKmsAlias = "arn:aws:kms:us-west-2:370957321024:alias/S3EC-Go-Github-KMS-Key"
```

Comment:
Question: Why change the bucket and key? Cause it got polluted with all the testing?

Reply:
The current defaults don't actually point to anything; this makes it so the defaults are the CI bucket/key so it's easier to test locally against the CI resources.